language-extensions/java/build/windows/create-java-extension-zip.cmd

```diff
@@ -19,7 +19,15 @@ SET BUILD_OUTPUT=%ENL_ROOT%\build-output\java-extension\windows\%BUILD_CONFIGURA
 SET OUTPUT_JAR="%ENL_ROOT%\build-output\java-extension\target\%BUILD_CONFIGURATION%\mssql-java-lang-extension.jar"
 
 mkdir %BUILD_OUTPUT%\packages
-powershell -NoProfile -ExecutionPolicy Unrestricted -Command "Compress-Archive -Path %BUILD_OUTPUT%\javaextension.dll, %OUTPUT_JAR% -DestinationPath %BUILD_OUTPUT%\packages\java-lang-extension.zip -Force"
+
+REM Check if BUILD_CONFIGURATION is debug, then include javaextension.pdb in the zip
+IF /I "%BUILD_CONFIGURATION%"=="debug" (
+    SET INCLUDE_FILES=%BUILD_OUTPUT%\javaextension.dll, %OUTPUT_JAR%, %BUILD_OUTPUT%\javaextension.pdb
```

Comment:
to avoid duplication, 
`%BUILD_OUTPUT%\javaextension.dll, %OUTPUT_JAR%` can be saved to another variable before the IF so it can be reused in both IF/ELSE.

Reply:
Resolved!